src/content/en/updates/2019/05/desynchronized.md

```diff
@@ -0,0 +1,189 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The desynchronized hint invokes bypasses the DOM to eliminate the latency caused by using the renderer compositor queue.
+
+{# wf_updated_on: 2019-05-02 #}
+{# wf_published_on: 2019-05-02 #}
+{# wf_tags: chrome75 #}
+{# wf_featured_image: /web/updates/images/2019/05/latency.png #}
+{# wf_featured_snippet: The desynchronized hint invokes bypasses the DOM to eliminate the latency caused by using the renderer compositor queue. #}
+{# wf_blink_components: Blink>Canvas #}
+
+# Low-latency rendering with the desynchronized hint {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+<img src="/web/updates/images/2019/05/latency.png" class="hint-image attempt-right" alt="Differences in stylus rendering">
+
+<style>
+.hint {
+  height: 150px;
+  width: auto;
+}
+</style>
+
+Stylus-based drawing applications built for the web have long suffered from
+latency issues because a web page has to synchronize graphics updates with the
+DOM. In any drawing application, latencies longer than 50 milliseconds can
+interfere with a user's hand-eye coordination, making applications difficult to
+use.   
+
+The `desynchronized` hint for `canvas.getContext()` invokes a different code
```

Comment:
Can we get a code sample on how to apply the hint?

Reply:
I took me a minute to realize why you didn't see the sample code. Please see the refactored headings starting at line 71.